.github/workflows/frontend_checks.yml

```diff
@@ -34,17 +34,16 @@ jobs:
         run: npm install
 
       - name: Lint
-        run: npm run lint
+        run: turbo run lint
```

Comment:
probably have to change this back to `npm` since gitlab runner doesn't have turbo